examples/v2/chat2.nim

```diff
@@ -55,15 +55,16 @@ proc initAddress(T: type MultiAddress, str: string): T =
     raise newException(ValueError,
                          "Invalid bootstrap node multi-address")
 
-# NOTE Dialing on WakuRelay specifically
-proc dialPeer(c: Chat, address: string) {.async.} =
+proc parsePeer(address: string): PeerInfo = 
   let multiAddr = MultiAddress.initAddress(address)
   let parts = address.split("/")
-  let remotePeer = PeerInfo.init(parts[^1], [multiAddr])
+  result = PeerInfo.init(parts[^1], [multiAddr])
 
-  echo &"dialing peer: {multiAddr}"
+# NOTE Dialing on WakuRelay specifically
+proc dialPeer(c: Chat, peer: PeerInfo) {.async.} =
+  echo &"dialing peer: {peer.peerId}"
   # XXX Discarding conn, do we want to keep this here?
-  discard await c.node.switch.dial(remotePeer, WakuRelayCodec)
+  discard await c.node.switch.dial(peer, WakuRelayCodec)
```

Comment:
So this is only calling on WakuRelayCodec, you might need something else for Store?

Reply:
no, because the WakuStore protocol dials by itself.